readme.md

```diff
@@ -135,41 +142,66 @@ var a = 2;
 
 ### Default parameters (5 / 40)
 
-With ES6, we now have the option add set a default value for any of our functions' parameters.
+If a function defined to take 3 parameters is called with two arguments, what is the value of the third parameter in the function body?
```

Comment:
Maybe this is an opportunity to be verbose for the sake of clarity.

`We've defined a function below with 3 parameters. If we invoke the function and only supply 2 arguments,`...